templates/stamp/utilities.sh

```diff
@@ -96,4 +96,34 @@ install-git()
         log "Installing Git Client"
         apt-get install -y git
     fi
+}
+
+#############################################################################
+# Install Mongo Shell
+#############################################################################
+
+install-mongodb-shell()
+{
+    if type mongo >/dev/null 2>&1; then
+        log "MongoDB Shell is already installed"
+    else
+        log "Installing MongoDB Shell"
+        apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv EA312927
```

Comment:
the jumbox will be version 12 so we'll need to conditionalize this. 

the implementation in mongodb-ubuntu-install.sh within the deployment branch is

https://github.com/Microsoft/oxa-tools/blob/deployment/templates/stamp/mongodb-ubuntu-install.sh

OS_VER=$(lsb_release -rs)
if (( $(echo "$OS_VER > 16" |bc -l) ))
then
    apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv EA312927
    echo "deb ${PACKAGE_URL} "$(lsb_release -sc)"/mongodb-org/3.2 multiverse" | tee /etc/apt/sources.list.d/mongodb-org-3.2.list
else
    apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 7F0CEB10
    echo "deb ${PACKAGE_URL} "$(lsb_release -sc)"/mongodb-org/3.0 multiverse" | tee /etc/apt/sources.list.d/mongodb-org-3.0.list
fi